src/amplify-migration/yamlUtils.ts

```diff
@@ -0,0 +1,115 @@
+import YAML, { Scalar, isPair, isScalar, isMap, isSeq, Pair } from "yaml";
+import { getRawPermalink } from "./jsDomUtils";
+import { REPOS_WITH_ERRORS } from "./constants";
+import os from "os";
+import fs from "fs";
+import path from "path";
+import { errorMessage } from "./errorMessage";
+
+export const YML_KEYS = [
+  "shareicon",
+  "favicon",
+  "file_url",
+  "image",
+  "logo",
+  "background",
+];
```

Comment:
Just for my understanding, these are the only yml keys which can possibly contain a media file right?

Reply:
yes, as far as I have seen uh